doc/api.rst

```diff
@@ -1,91 +1,107 @@
-DeePMD-kit API
-===============
+API
+===
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents:
 
-.. automodule:: deepmd.Data
+.. automodule:: deepmd.utils.data
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DataModifier
+.. automodule:: deepmd.infer.data_modifier
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DataSystem
+.. automodule:: deepmd.utils.data_system
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DeepDipole
+.. automodule:: deepmd.infer.deep_eval
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DeepEval
+.. automodule:: deepmd.infer.deep_polar
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DeepPolar
+.. automodule:: deepmd.infer.deep_pot
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DeepPot
+.. automodule:: deepmd.infer.deep_wfc
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DeepWFC
+.. automodule:: deepmd.descriptor.loc_frame
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DescrptLocFrame
+.. automodule:: deepmd.descriptor.se_a
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DescrptSeA
+.. automodule:: deepmd.descriptor.se_ar
    :members:
    :undoc-members:
 
-.. automodule:: deepmd.DescrptSeAR
+.. automodule:: deepmd.descriptor.se_r
```

Comment:
I think some other packages are missing here?

Reply:
Indeed. Thank you.